### ISSUE TITLE
FrameSamplingFilter shows same frame for last 1/2 of frames.

### ISSUE THREAD

**Issue by adelavega** [PsychoinformaticsLab/pliers#291]
A weird bug that is likely taking place moviepy or ffmpeg.

I ran a movie through `FrameSamplingFilter` and the last half of the frames were actually the same frame, with different onsets. 

When I transcoded the movie clip, the issue went away, leading me to think its a weird issue with the codec.

**Comment by tyarkoni** [comment]
Seems like there's a good chance #298 is responsible for this, so just linking it here.

**Comment by tyarkoni** [comment]
I think the sense from @qmac is that this is a bug in ffmpeg or moviepy (or reflects the interaction between the two). Will leave it open in case other people run into this, but short of a moviepy fix, there's probably not much we can do.

**Comment by qmac** [comment]
Yes, and it is worth noting that this issue is completely independent of pliers and the `FrameSamplingFilter`. This happens when iterating over the frames of the moviepy clip, and seems to be a result of ffmpeg misreading the codec information and moviepy subsequently skipping too many frames as a result of [this block of code](https://github.com/Zulko/moviepy/blob/master/moviepy/video/io/ffmpeg_reader.py#L178)

**Comment by tyarkoni** [comment]
@adelavega do you know if this is still a problem?

**Comment by adelavega** [comment]
I'm not sure, it was only an issue with a specific codec. I'll see if I can test on the same file if I can find it...

**Comment by adelavega** [comment]
Looks like it, but not quite as bad. The last 2000 frames were the same out of 20000

**Comment by tyarkoni** [comment]
Okay, will continue to leave this open. Thanks for checking!
